src/main/java/module-info.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Module dependencies.
+ */
+module MagmaCore.service.main {
```

Comment:
It defines the dependencies on other modules and will be needed later. I would suggest changing the module name to something like `uk.gov.gchq.magmacore` at that point as I believe that is more like the recommended naming convention.

Reply:
But until Magma Core actually becomes a Java module these dependencies are defined by the pom.xml? I'd rather we only include this file once it really becomes necessary. Otherwise it could be misleading when looking back through the version history.

Reply:
It already is a Java 9 module (I'm using it as such in another prototype), but it's not yet a maven module.

Reply:
Okay, we'll leave this as-is until properly modularised then.